tests/foreman/api/test_convert2rhel.py

```diff
@@ -53,34 +55,21 @@ def create_activation_key(sat, org, lce, cv, subscription_id):
 
 def update_cv(sat, cv, lce, repos):
     """Update and publish Content view with repos"""
-    cv = sat.api.ContentView(id=cv.id, repository=repos).update(["repository"])
+    cv = sat.api.ContentView(id=cv.id, repository=repos).update(['repository'])
     cv.publish()
     cv = cv.read()
     cv.version.sort(key=lambda version: version.id)
     cv.version[-1].promote(data={'environment_ids': lce.id, 'force': False})
     return cv
 
 
-def register_host(sat, act_key, org, module_loc, host, ubi=None):
-    """Register host to satellite"""
-    # generate registration command
-    command = sat.api.RegistrationCommand(
-        organization=org,
-        activation_keys=[act_key.name],
-        location=module_loc,
-        insecure=True,
-        repo=ubi,
-    ).create()
-    assert host.execute(command).status == 0
-
-
 @pytest.fixture(scope='module')
-def ssl_cert(module_target_sat, module_org):
+def ssl_cert(module_target_sat, module_entitlement_manifest_org):
     """Create credetial with SSL cert for Oracle Linux"""
     res = requests.get(settings.repos.convert2rhel.ssl_cert_oracle)
     res.raise_for_status()
     return module_target_sat.api.ContentCredential(
-        content=res.text, organization=module_org, content_type='cert'
+        content=res.text, organization=module_entitlement_manifest_org, content_type='cert'
```

Comment:
Good point, I'm looking to cherrypick this fix until 6.12.z where this is still supported, and this will be removed in  latest stream, so I'd open a separate PR changing it later, thank you!

Reply:
I've updated the same PR to include those as SCA is enabled as default on stream